test/fixtures/iotFleetProvisioning/registerDevice.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+// NOTE: `aws-iot-device-sdk-v2` is bundled into the deployment package
+// eslint-disable-next-line import/no-unresolved
+const { mqtt, io, iot, iotidentity } = require('aws-iot-device-sdk-v2');
+
+const subscriptionCallback = (error, response, channel, resolve, reject) => {
+  if (response) {
+    console.log(`${channel} successful message: ${JSON.stringify(response)}`);
+  }
+
+  if (error || !response) {
+    console.log(`Error occurred on channel ${channel}.`);
+    reject(error);
+  } else {
+    resolve(response);
+  }
+};
+
+const executeKeys = async identity => {
+  return new Promise((resolve, reject) => {
+    console.log('Subscribing to CreateKeysAndCertificate Accepted and Rejected topics..');
+
+    const keysSubRequest = {};
+
+    return Promise.all([
```

Comment:
Return values of promise executors are ignored (I think we should have lint rule, that will pick that, but unfortunately there's not)

This construct creates orphaned (not handled by any logic) promise, which is error-prone, we definitely should not do that